source/Calamari.Common/Plumbing/Variables/ScriptVariables.cs

```diff
@@ -26,6 +26,11 @@ public static string GetLibraryScriptModuleName(string variableName)
         {
             return variableName.Replace("Octopus.Script.Module[", "").TrimEnd(']');
         }
+        
+        public static string FormatScriptName(string scriptName)
+        {
+            return new string(scriptName.Where(c => char.IsLetterOrDigit(c) || c == '_').ToArray());
```

Comment:
is there any other "normal" chars we should support? `-` for instance?

Reply:
That's the kind of thing I'm trying to understand as I think that largely depends on intended behaviour and why we originally only limited this to letters/digits. Its one of those [Chesterton's Fence](https://fs.blog/chestertons-fence/) situations.

If this is only an arbitrary limitation, arguably we should include `-`, but then theres the question of whitespace and other characters (don't quote me on it, but I believe `!@#$%^&()` are all valid on Win/Unix systems).